packages/gestalt/src/Tabs.js

```diff
@@ -117,16 +136,24 @@ export const TabWithForwardRef: AbstractComponent<TabProps, HTMLElement> = forwa
         onMouseUp={() => setPressed(false)}
         onMouseEnter={() => setHovered(true)}
         onMouseLeave={() => setHovered(false)}
-        onTap={({ event, disableOnNavigation }) =>
-          onChange({ activeTabIndex: index, event, disableOnNavigation })
-        }
+        onTap={({ event, disableOnNavigation }) => {
+          onChange({ activeTabIndex: index, event, disableOnNavigation });
+        }}
         role="link"
         rounding={TAB_ROUNDING}
-        tapStyle="compress"
+        tapStyle={isActive ? 'none' : 'compress'}
       >
         <Flex alignItems="center" direction="column">
           <Box
-            color={color}
+            // $FlowExpectedError[incompatible-type] Flow doesn't understand the non-RGBA colors are valid for Box
+            color={color.startsWith('rgba') ? undefined : color}
```

Comment:
Are the colors allowed in Box exported? Just thinking it might be easier to make the logic read more like "If it's a color Box accepts, use it, otherwise set it as a backgroundColor" instead of replying on it starting with "rgba"

Reply:
They aren't currently, but that's a good idea!

Reply:
Dang, no dice. ☹️ Flow still complains.